tests/snapshots/mutual_recursion__a_b_c.hvm.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: tests/golden_tests.rs
+input_file: tests/golden_tests/mutual_recursion/a_b_c.hvm
+---
+Mutual recursion cycle detected in compiled functions:
+Cycle 1: A -> B -> C
```

Comment:
Can you make it so that we get the first element at the end of the list as well? In this case `A -> B -> C -> A`